### PR TITLE
MVP: Display prompts on cards

### PR DESCRIPTION
**Pull request by bfollington**
MVP of prompt cards. Choosing blindly at random from a small list of prompts and applying those to every card.

Simulator Screenshot - iPhone 15 Pro - 2023-12-22 at 14 25 10 Simulator Screenshot - iPhone 15 Pro - 2023-12-22 at 14 25 28

### CODE REVIEW

File: xcode/Subconscious/Shared/Models/Prompt.swift [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        Prompt(message: "How would someone else see this?"),
+        Prompt(message: "What problem can this idea solve?"),
+        Prompt(message: "Can you capture it in a metaphor?"),
+        Prompt(message: "Can it be simplified?"),
+        Prompt(message: "What are the potential consequences?"),
+        Prompt(message: "What are the ethical implications?"),
+        Prompt(message: "What happens if you change the context?"),
+        Prompt(message: "What are the unintended effects?"),
+        Prompt(message: "What would be a counter-argument?")
+    ]
```
Comment (bfollington) [inline xcode/Subconscious/Shared/Models/Prompt.swift:28]: Please feel free to change and extend these.

File: xcode/Subconscious/Shared/Models/Prompt.swift [hunk 2]
```diff
@@ -0,0 +1,68 @@
+//
+//  Prompt.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 13/12/2023.
+//
+
+import Foundation
+
+struct Prompt {
```
Comment (gordonbrander) [inline xcode/Subconscious/Shared/Models/Prompt.swift:10]: Nit: should be public if property is.

**Review by gordonbrander — APPROVED** [review]
LGTM. A couple of minor nits.

File: xcode/Subconscious/Shared/Components/Deck/CardStackView.swift [hunk 3]
```diff
@@ -34,37 +83,31 @@ struct CardView: View {
...
-                            Text("\(backlinks.count)")
-                        }
+                        Text(message)
                     }
+                    .padding(DeckTheme.cardPadding)
+                    .foregroundStyle(highlight)
+                    .font(.subheadline)
+                    
+                    Divider()
+                        .overlay(highlight)
```
Comment (gordonbrander) [inline xcode/Subconscious/Shared/Components/Deck/CardStackView.swift:103]: Nit: this divider looks heavy to my eye. I think it should visually read as a crease in the paper.

**Review by gordonbrander — APPROVED** [review]

**Pull request merged**